coda-automation/deb/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:stretch
```

Comment:
Thinking everything under `deb/` is also CRUFT tho someone correct me if I'm wrong. Also remove the keys file above?